apps/web/pages/settings/my-account/general.tsx

```diff
@@ -117,151 +118,152 @@ const GeneralView = ({ localeProp, user }: GeneralViewProps) => {
   } = formMethods;
   const isDisabled = isSubmitting || !isDirty;
 
+  const [isAllowDynamicBookingChecked, setIsAllowDynamicBookingChecked] = useState<boolean>(
+    user.allowDynamicBooking ?? true
+  );
+  const [isAllowSEOIndexingChecked, setIsAllowSEOIndexingChecked] = useState<boolean>(
+    user.allowSEOIndexing ?? true
+  );
+  const [isReceiveMonthlyDigestEmailChecked, setIsReceiveMonthlyDigestEmailChecked] = useState<boolean>(
```

Comment:
NIT: Not blocking

Lets try to use type infernace here - Normally explicity defining types is useful in cases where we're not sure what the type is - but given all of these are prefixed with `is` i think its fine to handle them as i suggested 
```suggestion
  const [isAllowDynamicBookingChecked, setIsAllowDynamicBookingChecked] = useState(
    !!user.allowDynamicBooking
  );
  const [isAllowSEOIndexingChecked, setIsAllowSEOIndexingChecked] = useState(
    user.allowSEOIndexing
  );
  const [isReceiveMonthlyDigestEmailChecked, setIsReceiveMonthlyDigestEmailChecked] = useState(
```